src/ardrive.ts

```diff
@@ -378,7 +378,7 @@ export class ArDrive extends ArDriveAnonymous {
 			owner
 		});
 
-		if (childrenFolderIds.includes(newParentFolderId)) {
+		if (childrenFolderIds.some((fid) => fid.equals(newParentFolderId))) {
```

Comment:
I didn't know this method. I assume it's similar to find, but doesn't ensure to return the 1st result

Reply:
It's as efficient but returns a boolean rather than an entry